pkg/controller/common/helpers.go

```diff
@@ -558,42 +559,27 @@ func removeIgnDuplicateFilesUnitsUsers(ignConfig ign2types.Config) (ign2types.Co
 // internally it transpiles to Ign spec v3 config
 func TranspileCoreOSConfigToIgn(files, units []string) (*ign3types.Config, error) {
```

Comment:
Is CoreOSConfig a subset of Butane? Could we rename this TranspileButaneToIgn? Or if there are differences maybe add a comment?

Reply:
CoreOSConfig came from https://github.com/coreos/container-linux-config-transpiler - it's very similar/related to butane, but there is no strict subset or superset relationship.

I have now clarified that these templates are using butane specifically `variant: fcos` `version: 1.2` which is defined to generate Ignition 3.2 which is what was output before.

(Really though, all we care about is the ability to conveniently define files inline)